src/Cli.Tests/AddTelemetryTests.cs

```diff
@@ -45,7 +45,7 @@ public void TestAddApplicationInsightsTelemetry(CliBool isTelemetryEnabled, stri
 
             // Add Telemetry
             bool isSuccess = ConfigGenerator.TryAddTelemetry(
-                new AddTelemetryOptions(appInsightsConnString, isTelemetryEnabled, configPath),
+                new AddTelemetryOptions(appInsightsConnString, isTelemetryEnabled, config: configPath),
```

Comment:
I think it would be better to specify all the parameters for consistency.
```suggestion
                new AddTelemetryOptions(appInsightsConnString: appInsightsConnString, appInsightsEnabled: isTelemetryEnabled, config: configPath),
```